foundry/loaders/tf_wrapper.py

```diff
@@ -24,4 +23,4 @@ def __getitem__(self, idx):
         item["target"] = np.array(item["target"])
         
         return item
-    
+        
```

Comment:
all of these files appear to be missing a proper newline at the end? please amend